src/components/JosekiAdmin/JosekiAdmin.tsx

```diff
@@ -407,9 +407,11 @@ export class JosekiAdmin extends React.PureComponent<JosekiAdminProps, JosekiAdm
                     ]}
                 />
                 <div className="explorer-stats">
-                    <span>Page visits: {this.state.page_visits || "..."}</span>
+                    <span>
+                        {_("Page visits")}: {this.state.page_visits || "..."}
+                    </span>
                     <button className="btn s" onClick={this.showVisitStats}>
-                        details
```

Comment:
We should use `pgettext` to provide some context,

`pgettext("Details about visit statistics", "details")` 

kind of thing

Reply:
Yeah I looked at that one and was lazy, mea culpa.

Reply:
Yeah, I've regretted being lazy about that in the past, the only reason why I'm being more pedantic about it now :) translators used to (and probably still do) have a hard time translating some things I was lazy about because they lacked context.. now I try to provide context for anything that's kind of generic like this.